core/elbv2.go

```diff
@@ -0,0 +1,37 @@
+package core
+
+import (
+	"github.com/aws/aws-sdk-go/service/elbv2"
+)
+
+// Returns a list of ELB (v2) - also known as ALB - based on the input from the different regions
+func (c *Connector) GetLoadBalancersV2(input *elbv2.DescribeLoadBalancersInput) ([]*elbv2.DescribeLoadBalancersOutput, error) {
```

Comment:
Because of https://github.com/cycloidio/raws/pull/1/files#r122167049

I would return and `Errs` type rather than an `error` interface, it isn't a problem for the caller because `Errs` satisfies the `error` interface